### PR TITLE
Adding support for multiple tables with same name across different databases

### PR DESCRIPTION
**Pull request by dasilva333**
Also as part of the change this need to be included as well:
https://github.com/dasilva333/sails-mysql/commit/ae67aedcd7793ceb2b7d9a9232af27dfa782ebfe

### CODE REVIEW

**Comment by tjwebb** [comment]
Can you document how this feature would work?

**Comment by dasilva333** [comment]
It seems there is an issue with waterline/sails-mysql/waterline-sequel to reproduce the problem try the following steps:

1. create a database called 'cats' in mysql
2. create another database called 'dogs' in mysql
3. create a table called owners in both databases.
4. Attempt to load those two models
5. Verify it does not work with and without the identity property on the model

The solution I came up with might not be the ideal solution because it required me to assign an identity property to every model before I load it like this:

modelObject.identity = modelObject.connection.toLowerCase()+"_"+modelObject.tableName.toLowerCase();

and then later to:

orm.loadCollection(Waterline.Collection.extend(modelObject));

Basically I'm submitting my solution but also suggesting someone else look further into this simple and reproducible problem with the code.

**Comment by dasilva333** [comment]
Also of note this ticket was also reported elsewhere on the parent project:
https://github.com/balderdashy/sails-mysql/issues/171

**Comment by kai23** [comment]
As reported on the parent project, adding an identity « works » great, but fail on populate.
